src/console/App.js

```diff
@@ -1,4 +1,7 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Map } from 'immutable';
```

Comment:
super-nit: imports should be alphabetical (`immutable`, `prop-types`, `react`, `react-redux`). We currently do not have an eslint rule for this (the particular plugin is in development iirc), so I imagine this will get past us in a few places anyway. Just something to keep in mind!